apisix/plugins/ext-plugin/init.lua

```diff
@@ -655,6 +655,12 @@ local rpc_handlers = {
                 end
             end
 
+            local len = rewrite:BodyLength()
+            if len > 0 then
+                let body = rewrite:BodyAsString()
+                ngx.req.set_body_data(body)
```

Comment:
```suggestion
                ngx.req.read_body()
                ngx.req.set_body_data(body)
```
We need call `ngx.req.read_body()` before `set_body_data`